netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -80,12 +96,23 @@
 
     @Override
     public Response act(final Request req) throws IOException {
-        final RqMultipart.Smart multi = new RqMultipart.Smart(
+        final Request file = new RqMultipart.Smart(
             new RqMultipart.Base(req)
+        ).single("file");
+        final Matcher matcher = FILE_NAME_PATTERN.matcher(
```

Comment:
@antonini if this constant is used once and only inside this method, maybe we can inline?